packages/manager/modules/pci/src/projects/new/payment/routing.js

```diff
@@ -178,7 +179,15 @@ export default /* @ngInject */ ($stateProvider) => {
         );
       },
 
-      step: /* @ngInject */ (getStep) => getStep('payment'),
+      onProgressStepClick: /* @ngInject */ ($state) => ({ name, active }) => {
```

Comment:
Could the button be disabled here https://github.com/ovh/manager/pull/6916/files#diff-ab7d179cf72d63bd70461919826103ce4f60cd0eb62237c5575d96ec80b28de0R10  instead of returning null if conditions are not verified?

Reply:
Yep, it could be an improvement but for the moment is fully working. 

Thanks